Makefile

```diff
@@ -169,17 +170,17 @@ fuzzertest: debug
 			--minloglevel=0
 
 format-fix: 			#: Fix formatting issues in the main branch
-	scripts/check.py format main --fix
+	source ${PYTHON_VENV}/bin/activate; scripts/check.py format main --fix
```

Comment:
What happens if the user hasn't run the setup script to create the venv in the velox repo dir?
As in they pick up this change and run the `format-fix`.